mqtt3/1.0.0/modules/ROOT/pages/mqtt3-documentation.adoc

```diff
@@ -0,0 +1,490 @@
+
+
+====== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Client ID a| String |  Client ID to identify the connection to the MQTT broker. |  | x
+| Username a| String |  Username to authenticate credentials to the MQTT broker. |  |
+| Password a| String |  Password authenticate credentials to the MQTT broker. |  |
+| Last Will and Testament (LWT) a| <<LwtMessage>> |  |  |
+| Connection Settings a| One of:
+
+* <<FormMqtt3Authenticator>>
+* <<ConnectionStringMqtt3Authenticator>> |  |  | x
+| TLS Context a| <<Tls>> |  |  |
+| Keep Alive Interval a| Number |  The amount of time that the connection between the client and broker is kept alive without any messages exchanged. |  60 |
+| Keep Alive Interval Unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  The unit of time that corresponds to the keepAliveInterval parameter. |  SECONDS |
+| Max In Flight a| Number |  The maximum amount of messages to unacknowledge at a given time. |  10 |
+| Clean Session a| Boolean |  If set to `true`, the session is cleaned each time the client disconnects from the broker. Subscriptions are not saved and offline (QoS 1 and 2) messages for that client are lost. |  `true` |
+| Enable File Persistence a| Boolean |  If set to `true`, creates an file-based persistent data store, used to store outbound and inbound messages while they are in flight, enabling delivery to the QoS specified. |  `false` |
+| Reconnection a| <<Reconnection>> |  When the application is deployed, a connectivity test is performed on all connectors. If set to `true`, deployment will fail if the test doesn't pass after exhausting the associated reconnection strategy |  |
+|===
+[[Config_Resilient]]
+===== MQTT3 Resilient Connection
+
+
+====== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Client ID a| String |  Client ID to identify the connection to the MQTT broker. |  | x
+| Username a| String |  Username to authenticate credentials to the MQTT broker. |  |
+| Password a| String |  Password to authenticate credentials to the MQTT broker. |  |
+| Last Will and Testament (LWT) a| <<LwtMessage>> |  |  |
+| Fail Over Servers a| Array of <<FailOverUrl>> |  The list of server URLs used to establish a connection to the broker. |  | x
+| TLS Context a| <<Tls>> |  |  |
+| Keep Alive Interval a| Number |  The amount of time that the connection between the client and broker is be kept alive without any messages exchanged. |  60 |
+| Keep Alive Interval Unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  The unit of time that corresponds to the keepAliveInterval parameter. |  SECONDS |
+| Max In Flight a| Number |  The maximum amount of messages that are unacknowledge at a given time. |  10 |
```

Comment:
Typo : are unacknowledge 

Elsewhere, you have this text, which is fine: The maximum amount of messages to unacknowledge at a given time.